### DISCUSSION TITLE
Connection issues with Pixel 6 Pro and WearOS watch

### DISCUSSION THREAD

**Discussion by cwpeterson** [NightscoutFoundation/xDrip#2016] [Q&A]
I've posted about this on the Facebook group some time back. Since getting my Pixel 6 Pro last fall I have connectivity issues with my Dexcom G6 sensor after restarting the phone. This has been happening across many XDrip versions. After another user mentioned that once he stopped using his WearOS watch, the issues went away. I have a Fossil Gen 5 watch. As soon as I power off the watch I have zero issues keeping connection to the sensor. Eventually I can get both working together with the phone after just letting it go for a while, sometimes 12+ hours and both will work fine. Until the next phone restart and then the battle starts all over again to get it to pull a reading. I can see that it DOES connect to the sensor, it just doesn't get a reading. On the system status screen it says "Bonding". I can force it to pull a reading if I restart the collector just before it reconnects to the sensor but then it won't get a reading for some time after. It's sporadic. Again it does eventually seem to resolve itself. However after the latest patch from Google that I installed yesterday it's worse than before. I had to leave the watch powered off overnight and as soon as I powered it on today it started having issues again. 

I really like my watch, my phone, and XDrip and don't want to switch anything. It's all worked so well to be able to get my readings on my watch.

Anyway, wasn't sure yet if I should open a bug report or if anyone out there has any ideas or troubleshooting steps to take this further to resolve it?

**Comment by Navid200** [comment]
Thanks for posting here.
There are already several open issues that are related: 
https://github.com/NightscoutFoundation/xDrip/issues/1789
https://github.com/NightscoutFoundation/xDrip/issues/1664

I suggest waiting (not opening an issue) a little bit to see if @jamorham may have any suggestions here.

**Reply by cwpeterson** [reply]
I looked but apparently not hard enough. Thanks.
